### ISSUE TITLE
Log when a user tries to open a media file but it's not on the phone

### ISSUE THREAD

**Issue by alexlittle** [DigitalCampus/oppia-mobile-android#539]
Probably best to add to the tracker again - but will need to think a little more about how it should all be specified. Not sure if also needs update on server side for the tracker API? Reason to do this is so we know if users are at least trying to watch the videos.

**Comment by jjoseba** [comment]
Yes, it would require a minimum validation from server side also. I've created the issue in the server side: https://github.com/DigitalCampus/django-oppia/issues/387
